h2o-core/src/test/java/water/fvec/FrameTest.java

```diff
@@ -55,10 +55,13 @@ public void testNonEmptyChunks() {
   @Test
   public void testRemoveColumn() {
     Scope.enter();
-    Frame testData = parse_test_file(Key.make("test_deep_select_1"), "smalldata/sparse/created_frame_binomial.svm.zip");
-    Set<Vec> removedVecs = new HashSet<>();
+    Set<Vec> removedVecs = null;
 
     try {
+      Frame testData = parse_test_file(Key.make("test_deep_select_1"), "smalldata/sparse/created_frame_binomial.svm.zip");
+      Scope.track(testData);
+      removedVecs = new HashSet<>();
```

Comment:
merge this with L58.
otherwise you can get a NPE in the finally block if you get an error during parsing.

Reply:
@seb-h2o thanks! I was thinking about it as well. I wanted even to introduce helper Scope.track method vararg of the vecs. But maybe for now there is a simpler solution -> see commit